config/volume.go

```diff
@@ -171,18 +214,20 @@ func (c *Client) ListVolumes(policy string) (map[string]*Volume, error) {
 	configs := map[string]*Volume{}
 
 	for _, node := range resp.Node.Nodes {
-		if node.Value == "" {
-			continue
-		}
+		if len(node.Nodes) > 0 {
+			node = node.Nodes[0]
+			if !node.Dir && strings.HasSuffix(node.Key, "/create") {
```

Comment:
if I am reading this correct, looks like ListVolumes() will now just show Create Config related info for a volume. Shouldn't we show both? I understand that we have a separate API to show runtime configuration, but I am wondering if we need a CLI or UX to show consolidated info as well.